### DISCUSSION TITLE
Can Chat cite sources?

### DISCUSSION THREAD

**Discussion by Ikealightbulb** [brianpetro/obsidian-smart-connections#294] [Ideas]
Thank you for this Plugin. 
Smart Connections is very helpful. Surfacing relevant notes, helping me link them. 
For the Chat function, I wish that it could also include sources (refer to specific notes), like Bing Chat. 
Thank you !

**Comment by RealRavens** [comment]
Linked notes being included or referenced in chat is an upcoming feature. I'm waiting for this as well. Smart connections is great, but with smart chat communicating with your notes, you could make connections on steroids.

[![Image from Gyazo](https://i.gyazo.com/57bcda17484b77a8d5964b81699a2619.png)](https://gyazo.com/57bcda17484b77a8d5964b81699a2619)
